androidApp/src/main/res/raw/sample_assessment.json

```diff
@@ -18,7 +18,40 @@
     {
       "identifier": "step1",
       "type": "instruction",
-      "title": "Step 1"
+      "title": "Instruction Step 1",
+      "detail": "Here are the details for this instruction.",
+      "image"  : {
+        "type":"fetchable",
+        "imageName" : "crf_seated"
+      }
+    },
+    {
+      "identifier": "step2",
+      "type": "instruction",
+      "title": "Animating image instruction",
+      "detail": "Here are the details for this instruction.",
+      "image"  : {
+        "type":"fetchable",
+        "imageName" : "crf_stair_step_start_animation"
+      }
```

Comment:
So... this won't work for iOS. An animating image must be defined with a series of image names using the `AnimatedImage` serializable class. This is not compatible JSON.

Reply:
On Android an AnimationDrawable is usually defined in a Drawable xml file which contains the list of images and any animation configuration. I suppose we could manually do that in code.